src/components/Orders/OrdersContainer.js

```diff
@@ -0,0 +1,117 @@
+                setOrdersList(orders);
+                setIsLoading(false);
+            });
+    }, []);
+
+    return (
+        <Flex h='100vh' py={20}>
+            <SimpleGrid columns={3} rowGap={6} columnGap={3} w='full'>
+                <GridItem colSpan={3}>
+                    <Box
+                        w='full'
+                        h='full'
+                        p={3}
+                        spacing={10}
+                        borderWidth='1px'
+                        borderRadius='16px'
+                    >
+                        <Heading p={4} mb={6}>
+                            Your Orders
+                        </Heading>
+                        {isLoading ? (
+                            <Text>still loading data</Text>
+                        ) : (
+                            ordersList.map((order, index) => {
+                                return (
+                                    <Box
+                                        key={index}
+                                        p={3}
+                                        mb={3}
+                                        spacing={10}
+                                        borderWidth='1px'
+                                        borderRadius='16px'
+                                    >
+                                        <SimpleGrid columns={3} spacing={10}>
+                                            <Box height='80px'>
+                                                <Text color='gray'>
+                                                    Order placed
+                                                </Text>
+                                                {/* order.created_at.getMonth(), order.created_at.getDate(), order.created_at.getFullYear() */}
+                                                {/* /{order.created_at.getDate()}/{order.created_at.getFullYear()} */}
+                                                <Text>
+                                                    {new Date(
+                                                        order.created_at
+                                                    ).toLocaleDateString(
+                                                        {},
+                                                        {
+                                                            timeZone: 'UTC',
+                                                            month: 'short',
+                                                            day: '2-digit',
+                                                            year: 'numeric',
+                                                        }
+                                                    )}
+                                                </Text>
+                                            </Box>
+                                            <Box height='80px'>
+                                                <Text color='gray'>Total</Text>
```

Comment:
If you're going to make changes after this review, try to use the `colors` const in `/src/util/constants.js`. There's a gray color you can use there.

Reply:
will change.